sklearn/datasets/_twenty_newsgroups.py

```diff
@@ -391,23 +394,39 @@ def fetch_20newsgroups_vectorized(*, subset="train", remove=(), data_home=None,
 
         .. versionadded:: 0.22
 
+    as_frame : bool, default=False
+        If True, the data is a pandas DataFrame including columns with
+        appropriate dtypes (numeric, string or categorical). The target is
+        a pandas DataFrame or Series depending on the number of target_columns.
+
+        .. versionadded:: 0.24
+
     Returns
     -------
     bunch : :class:`~sklearn.utils.Bunch`
         Dictionary-like object, with the following attributes.
 
         data: sparse matrix, shape [n_samples, n_features]
             The data matrix to learn.
+            If ``as_frame`` is True, ``data`` is a pandas object.
         target: array, shape [n_samples]
             The target labels.
         target_names: list, length [n_classes]
             The names of target classes.
+            If ``as_frame`` is True, ``target`` is a pandas object.
         DESCR: str
             The full description of the dataset.
 
     (data, target) : tuple if ``return_X_y`` is True
 
         .. versionadded:: 0.20
+
+    frame : pandas DataFrame
+        Only present when `as_frame=True`. DataFrame with ``data`` and
+        ``target``.
+
+        .. versionadded:: 0.24
```

Comment:
the `frame` is a part of the `Bunch` object

Reply:
Indeed, thanks. Copied this from the housing dataset, I'll open a PR to fix it there, too.